.ci/cico_updates_openshift.sh

```diff
@@ -25,7 +25,7 @@ overrideDefaults() {
 }
 
 runTests() {
-  "${OPERATOR_REPO}"/olm/testUpdate.sh "openshift" "stable" ${NAMESPACE}
+  "${OPERATOR_REPO}"/olm/testUpdate.sh -p openshift -c stable -i quay.io/eclipse/eclipse-che-openshift-opm-catalog:test -n ${NAMESPACE}
```

Comment:
quay.io/eclipse/eclipse-che-openshift-opm-catalog:test - I think we have to use catalog image which was build on the Openshift ci. I don't think it start from quay.io.

Reply:
OpenShift CI doesn't build any catalog sources in case of testing updates.
We use the existed one that is built during release.